test/index.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const Bookshelf = require('bookshelf')(require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  }
+}));
```

Comment:
I love the idea of using SQLite in this case, and I think having a test.sqlite3 file is fine.